### ISSUE TITLE
Should we actually include totalSupply?

### ISSUE THREAD

**Issue by metachris** [CityOfZion/neo-tokens#17]
Currently the token list includes total supply. Is this used for something? Wondering if it's a good idea, because some tokens have a variable total supply that increases over time (eg. red pulse).

**Comment by corck** [comment]
Same here, for our Gold token total supply doesn't make sense as we mint as soon as we story more Gold.

**Comment by mhuggins** [comment]
Just to add a little history here, there was a [nightly job that ran on CircleCI](https://circleci.com/gh/cityofzion/neo-tokens) to regularly update these values.  However, I think @comountainclimber disabled them due to some issues with the notification server intermittently having wrong/bad data.

**Comment by comountainclimber** [comment]
Yeah that is correct... There was an incident where the job just copied over an empty list causing some serious problems. If we could get some test coverage in that would solve this problem
